bikeshed/boilerplate.py

```diff
@@ -305,10 +305,10 @@ def addIndexSection(doc):
         container, E.h2({"class": "no-num no-ref", "id": safeID(doc, "index")}, "Index")
     )
 
-    if len(findAll(config.dfnElementsSelector, doc)):
+    if not findAll(config.dfnElementsSelector, doc):
         addIndexOfLocallyDefinedTerms(doc, container)
 
-    if len(list(doc.externalRefsUsed.keys())):
+    if not list(doc.externalRefsUsed.keys()):
```

Comment:
Feels like this could be unwrapped further to just `if not doc.externalRefsUsed:` but didn't want to play around too much

Reply:
I suspect this was part of the py2->3 migration tool, where a bunch of things that switched from lists to generators got manually coerced to lists for safety.